### PR TITLE
Scrawford dev

### PR DESCRIPTION
**Pull request by SCrawford51**
Adding base files copied from the iCache version of each module. While I left the ports and signals (with minor changes), the content of the module is empty. The purpose of this pull request is to make the bare files accessible to each of us as we start the project and to provide some experience with PRs.

### CODE REVIEW

**Comment by SCrawford51** [comment]
As you can see, this PR can't be merged until all three of you approve. Currently, I only have two people assigned. If we find that getting approval from three people quickly is too difficult, I can change this to two. I would prefer not to go fewer than that. Once Eric accepts the email to be a collaborator I will add him to this PR.

**Review by silva237 — APPROVED** [review]
Approved!

**Review by apierson5 — APPROVED** [review]

**Comment by SCrawford51** [comment]
I added the rest of the contents of neorve32_dcache.vhd and pulled down the latest updates from the parent repo.

**Review by silva237 — APPROVED** [review]

**Review by Eglynn3 — APPROVED** [review]

**Review by silva237 — APPROVED** [review]
Looks Good. Identical to icache files.

**Pull request merged**
